ingreedytest.py

```diff
@@ -78,6 +102,38 @@
         }],
         'ingredient': 'potatoes',
     },
+    '12 gram(s) potatoes': {
+        'quantity': [{
+            'amount': 12,
+            'unit': 'gram',
+            'unit_type': 'metric',
+        }],
+        'ingredient': 'potatoes',
+    },
+    '12gram(s) potatoes': {
+        'quantity': [{
+            'amount': 12,
+            'unit': 'gram',
+            'unit_type': 'metric',
+        }],
+        'ingredient': 'potatoes',
+    },
+    '12G. potatoes': {
+        'quantity': [{
+            'amount': 12,
+            'unit': 'gram',
+            'unit_type': 'metric',
+        }],
+        'ingredient': 'potatoes',
+    },
+    '12G potatoes': {
```

Comment:
In this case _perhaps_ it's OK to make an exception (as compared to `cups / C`) for case-sensitivity around gram units.